internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +574,33 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_tracing) {
+    echo "Downloading $dotnet_tracing_dl_url..."
+    download_file -url "$dotnet_tracing_dl_url" -outputDir "$tempdir" -fileName "$dotnet_tracing_msi_name"
+    $dotnet_tracing_msi_path = (Join-Path "$tempdir" "$dotnet_tracing_msi_name")
+
+    echo "Installing $dotnet_tracing_msi_path ..."
+    Start-Process msiexec.exe -Wait -ArgumentList "/qn /norestart /i `"$dotnet_tracing_msi_path`""
+    echo "- Done"
+
+    update_registry -path "$regkey" -name "COR_ENABLE_PROFILING" -value "1"
```

Comment:
Isn't it going to enable profiling globally? (machine level variable)
That could cause crashes and even disable remote login, due there are many local applications built on .NET. Specially it doesn't work well with PowerShell (which is also .NET based).

Reply:
Yes, definitely -- the idea is to have the customer just install one thing and auto-instrumentation just work ("zero config"). Can there be, or is there, a filter in the .NET library to not instrument some applications?

Reply:
https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/advanced-config.md#global-management-settings
`SIGNALFX_PROFILER_EXCLUDE_PROCESSES`

Unfortunately we didn't work out whole list, what should be disabled. PowerShell, Server Manager, IIS Manager are the first ones that come to my mind.

Reply:
+1 the ability to exclude processes from profiling is important. Similarly, is there a way to filter processes for trace instrumentation?

Reply:
Yes, we didn't separate the logic, 1 env works for all.

Reply:
@aunshc @pmcollins @RassK we have to be careful with the term "profiling" in the .NET context. When we say "profiling" we mean "a .NET profiler" or "CLR profiler" that is a term to identify a piece of code that attaches to a .NET process that has capabilities of doing bytecode instrumentation. We also use the same technology to do the actual "Always on Profiling", ie.: thread sampling/CPU profiling/Memory profiling, features The `COR_ENABLE_PROFILING` above is about attaching a ".NET Profiler" to the process if it will generate traces, metrics, or thread sampling data is controlled by other settings.

To @aunshc point: yes, it is possible to enable/disable "always profiling" separately from distributed tracing settings, see https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/always-on-profiling.md#configuration-settings

Reply:
Setting `COR_ENABLE_PROFILING` globally can bring catastrophic side-effects.

Lots of processes on Windows are .NET. It leads to instrumenting too much. 

At some point, we changed our docs to make sure that one does NOT enable it globally and also added a troubleshooting topic: https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/troubleshooting.md#high-cpu-usage

We are not setting these env vars in the MSI on purpose

Reply:
We can also consider setting these if the user provides process names that should be instrumented via `SIGNALFX_PROFILER_PROCESSES` (allowlist). 

Getting a proper value for `SIGNALFX_PROFILER_EXCLUDE_PROCESSES` (blocklist) may be hard. But maybe this is the way to go... We could achieve "zero-config" this way.

Reply:
Got it, thanks for the explanation! Good to know that there is a separate way to enable/disable always on profiling. Is there a way to associate the value of this setting to a specific .NET process vs the host globally?

Reply:
@aunshc This is not a way to disable "always on profiling" but the whole auto-instrumentation 😬 

[Here](https://github.com/signalfx/signalfx-dotnet-tracing/blob/v0.2.9/docs/advanced-config.md#alwayson-profiling-settings) are the settings for AlwaysOnProfiling if you are interested.

Reply:
Yep, with environment variables you can choose the right scope.... but the smaller the scope gets, the harder is to inject these variables. It's easy to setup global variables, but then these values can leak to unwanted processes. 

Why is it hard to have a small scope? - because mostly you just don't manually execute server's service. The simple .exe / .dll might be there but it's usually started by 3rd party: eg a webserver (IIS), scheduler, service watch etc. So you have to locate the master service and inject a config into startup.

Reply:
Yes @pellared ,makes sense - was referring to Paulo's comment above with https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/always-on-profiling.md#configuration-settings.